react/features/base/connection/actions.native.js

```diff
@@ -180,13 +183,13 @@ function _connectionDisconnected(connection, message) {
  *
  * @param {JitsiConnection} connection - The JitsiConnection which was
  * established.
- * @private
  * @returns {{
  *     type: CONNECTION_ESTABLISHED,
  *     connection: JitsiConnection
  * }}
+ * @public
```

Comment:
@emcho Sure! I was just expressing my personal preference for the new rule. But I don't agree that this is related to React especially if at some point we try to create a common JS code style guide for all our JS projects.